packages/storage/src/implementation/constants.ts

```diff
@@ -18,49 +18,50 @@
  * @fileoverview Constants used in the Firebase Storage library.
  */
 
+/**
+ * Domain name for firebase storage.
+ */
+export const DOMAIN_NAME: string = 'firebasestorage.googleapis.com';
+
 /**
  * Domain and scheme for API calls.
  */
-export const domainBase: string = 'https://firebasestorage.googleapis.com';
+export const API_DOMAIN: string = `https://${DOMAIN_NAME}`;
 
 /**
  * Domain and scheme for object downloads.
  */
-export const downloadBase: string = 'https://firebasestorage.googleapis.com';
+export const API_DOWNLOAD_DOMAIN: string = `https://${DOMAIN_NAME}`;
 
 /**
  * Base URL for non-upload calls to the API.
  */
-export const apiBaseUrl: string = '/v0';
+export const API_BASE_URL: string = '/v0';
 
 /**
  * Base URL for upload calls to the API.
  */
-export const apiUploadBaseUrl: string = '/v0';
-
-export function setDomainBase(domainBase: string) {
-  domainBase = domainBase;
-}
+export const API_UPLOAD_BASE_URL: string = '/v0';
 
 export const configOption: string = 'storageBucket';
 
 /**
  * 1 minute
  */
-export const shortMaxOperationRetryTime: number = 1 * 60 * 1000;
+export const SHORT_MAX_OPERATION_RETRY_TIME: number = 1 * 60 * 1000;
```

Comment:
Since these are now global constants, we should use much more descriptive names. Can this name be changed to reflect better what it is used for?